boards/arm64/qemu_cortex_a53/Kconfig

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2021 BayLibre SAS
+# SPDX-License-Identifier: Apache-2.0
+
+config HEAP_MEM_POOL_SIZE
+	default 1048510 if GIC_V3_ITS
```

Comment:
this is a pretty specific number :)

Reply:
I used 1Mib to be large, but I need to recalculate the exact memory needed

Reply:
Here are the system heap pool allocations:
- after gicv3 + its probe:
```
327408 free bytes, 720928 allocated bytes, overhead = 168 bytes (0.0%)
```
- after full test run (256 devices with 32 ITEs)
```
190192 free bytes, 854048 allocated bytes, overhead = 4264 bytes (0.4%)
```

The drivers allocation needs are (for QEMU):
- LPI prop table: global 1x64K aligned on 64K
- LPI pend table: for each redistributor/cpu 1x64K aligned on 64K
- Devices table: 8x64K aligned on 64K
- Interrupt Collections table: 1x64K aligned on 64K

This makes 11x64K allocations aligned on 64K, but due to aligment constrains, HEAP needs to be 14x64K.

The test allocation needs are (for QEMU):
- Interrupt Translation Table: 512 bytes x 256 Devices, aligned in 256

This makes 128K, but with aligment margins 15x64K fullfills all requirements.

I added this in comments and added the minimal needed in the boad Kconfig and extended the test with a board specific prj.conf